core/services/nmea_injector/settings.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict
+
+import pykson  # type: ignore
+from commonwealth.settings import settings
+
+
+class NmeaInjectorSettingsSpecV1(pykson.JsonObject):
+    kind = pykson.StringField()
+    port = pykson.IntegerField()
+    component_id = pykson.IntegerField()
+
+    @staticmethod
+    def from_spec(spec: "NmeaInjectorSpec") -> "NmeaInjectorSettingsSpecV1":  # type: ignore
+        return NmeaInjectorSettingsSpecV1(
+            kind=spec.kind,
+            port=spec.port,
+            component_id=spec.component_id,
+        )
+
+    def __eq__(self, other: object) -> Any:
+        if isinstance(other, NmeaInjectorSettingsSpecV1):
+            return self.kind == other.kind and self.port == other.port
+        return False
+
+
+class SettingsV1(settings.BaseSettings):
+    VERSION = 1
+    specs = pykson.ObjectListField(NmeaInjectorSettingsSpecV1)
```

Comment:
isn't it possible to remove specs and have the `NmeaInjectorSettingsSpecV1` properties here directly ?

Reply:
I don't think so, since this is where we store the multiple individual settings.